components/scripts/lib/info.sh

```diff
@@ -173,7 +204,9 @@ print_build_caching_leverage_metrics() {
     value="${taskCount} ${BUILD_TOOL_TASK}s, ${executed_not_cacheable_duration[1]} total execution time"
   fi
   summary_row "Executed non-cacheable ${BUILD_TOOL_TASK}s:" "${value}"
+}
 
+print_executed_cacheable_tasks_warning() {
   if (( executed_cacheable_num_tasks[1] > 0)); then
     print_bl
     warn "Not all cacheable ${BUILD_TOOL_TASK}s' outputs were taken from the build cache in the second build. This reduces the savings in ${BUILD_TOOL_TASK} execution time."
```

Comment:
0: I wonder if we should update this message to use the "build time savings" language:

```suggestion
    warn "Not all cacheable ${BUILD_TOOL_TASK}s' outputs were taken from the build cache in the second build. This reduces the build time savings in ${BUILD_TOOL_TASK} execution time."
```

Or even:

```suggestion
    warn "Not all cacheable ${BUILD_TOOL_TASK}s' outputs were taken from the build cache in the second build. This reduces the build time savings."
```

Reply:
Good suggestions! I will leave this decision to @etiennestuder.

Reply:
+1